docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-generic-oidc.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Configure Generic OIDC
+description: Create a OpenID Connect (OIDC) client and configure Rancher to work with your authentication provider. By the end your users will be able to sign into Rancher using their login from the authentication provider
+---
+
+<head> 
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-generic-oidc"/>
+</head>
+
+If your organization uses an OIDC provider for user authentication, you can configure Rancher to allow your users to log in using their IdP credentials. Rancher supports integration using the OpenID Connect (OIDC) protocol and the SAML protocol. Both implementations are functionally equivalent when used with Rancher. This page describes the process to configure Rancher to work using the OIDC protocol.
+
+## Prerequisites
+
+- On Rancher, Generic OIDC is disabled.
+- In your authentication provider, create a new client with the settings below (consult the documentation for your specific provider).
+
+     Setting | Value
+       ------------|------------
+     `Client ID` | &lt;CLIENT_ID> (e.g. `rancher`)
+     `Name` | &lt;CLIENT_NAME> (e.g. `rancher`)
+     `Client Protocol` | `openid-connect`
+     `Access Type` | `confidential`
+     `Valid Redirect URI` | `https://yourRancherHostURL/verify-auth`
+
+- In the new OIDC client, create mappers to expose the users fields (consult the documentation for your specific provider).
+  - Create a new "Groups Mapper" with the settings below.
+
+    Setting | Value
+        ------------|------------
+    `Name` | `Groups Mapper`
+    `Mapper Type` | `Group Membership`
+    `Token Claim Name` | `groups`
+    `Add to ID token` | `OFF`
+    `Add to access token` | `OFF`
+    `Add to user info` | `ON`
+
+  - Create a new "Client Audience" with the settings below.
+
+    Setting | Value
+        ------------|------------
+    `Name` | `Client Audience`
+    `Mapper Type` | `Audience`
+    `Included Client Audience` | &lt;CLIENT_NAME>
+    `Add to access token` | `ON`
+
+  - Create a new "Groups Path" with the settings below.
+
+    Setting | Value
+        ------------|------------
+    `Name` | `Group Path`
+    `Mapper Type` | `Group Membership`
+    `Token Claim Name` | `full_group_path`
+    `Full group path` | `ON`
+    `Add to user info` | `ON`
+
+ - Important:  Rancher will use the value received in the "sub" claim to form the PrincipalID which is the unique identifier in Rancher.  It is important to make this a value that will be unique and immutable.
```

Comment:
This line is indented 1 space. Is it intended to be a on the same level (no indentation) as line 25 `In the new OIDC client...` or is it supposed to be the same (2 spaces) as line 46 `Create a new "Groups Path" with the settings below.` ?